content/vault/v1.20.x/content/docs/audit/schema.mdx

```diff
@@ -0,0 +1,379 @@
+  "mount_class": "",
+  "mount_point": "",
+  "mount_type": "",
+  "mount_running_version": "",
+  "mount_running_sha256": "",
+  "mount_is_external_plugin": "",
+  "namespace": {
+    "id": "",
+    "path": ""
+  },
+  "operation": "",
+  "path": "",
+  "policy_override": true,
+  "remote_address": "",
+  "remote_port": 1234,
+  "replication_cluster": "",
+  "request_uri": "",
+  "wrap_ttl": 60
+}
+```
+
+## Response object (response) ((#response))
+
+The following response object definition includes example data with simple types (`string`, `bool`, `int`) and used in other JSON examples that include a `<response>` object.
+
+| Attribute | Type | Description |
+| :---- | :---- | :---- |
+| `auth` | `object` | If the request results in a token being created, an `auth` object with the token information. See [`auth`](#auth) above. |
+| `headers` | `object` | Set of HTTP response headers sent by the plugin that handled the API request. |
+| `redirect` | `string` | For requests sent to authentication backends, the URL to which the authentication backend redirected the user for further authentication. |
+| `warnings` | `list(string)` | If the API returned one or more warnings, a list of the warning messages. |
+| `data` | `object` | JSON object containing the response payload. This is dependent on the API endpoint called. |  |
+| `secret` | `object` | If the API returned a [leased secret](/vault/docs/concepts/lease), a JSON object with one attribute, `lease_id`, that identifies that leased secret. |
+| `wrap_info` | `object` | If the API returned a [response-wrapped secret](/vault/docs/concepts/response-wrapping), a JSON object containing the properties of the wrapping token. See below for the attributes in the JSON object. |
+| `mount_class` | `string` | Type of API mount, `auth` or `secret`. |
+| `mount_accessor` | `string` | Unique identifier of the Vault mount (secret engine or authentication backend) that responded to the API request. |
+| `mount_type` | `string` | Type of Vault mount that responded to the API request. |
+| `mount_running_plugin_version` | `string` | Version string for the Vault mount that responded to the API request. |
+| `mount_running_sha256` | `string` | If the mount that received the API request is an external plugin, the SHA-256 hash of the plugin's running version. |
+| `mount_is_external_plugin` | `boolean` | Whether the mount that received the API request is an external plugin. |
+
+```json
+{
+  "auth": <auth>,
+  "data": {},
+  "headers": {},
+  "mount_accessor": "",
+  "mount_class": "",
+  "mount_is_external_plugin": false,
+  "mount_point": "",
+  "mount_running_sha256": "",
+  "mount_running_plugin_version": "",
+  "mount_type": "",
+  "redirect": "",
+  "secret": {
+    "lease_id": ""
+  },
+  "wrap_info": {
+    "accessor": "",
+    "creation_path": "",
+    "creation_time": "",
+    "token": "",
+    "ttl": 60,
+    "wrapped_accessor": ""
+  },
+  "warnings": [
+    ""
+  ]
+}
+```
+
+## JSON schemas
```

Comment:
I'm not sure what this adds? We've basically duplicated the info in these partials in the tables and examples. What is the goal of this section? To be clear, I'm not against including it, I just don't understand the purpose